media/css/mozorg/about.less

```diff
@@ -0,0 +1,384 @@
+            margin:0 auto;
+        }
+
+        p {
+            margin: 10px 10px @baseLine;
+            .font-size(.857em);
+            font-style: italic;
+        }
+
+        .placeholder-img {
+            margin-bottom: @baseLine;
+        }
+
+        .moz-video-button {
+            background-image: url('/media/img/mozorg/about/poster-i-am.jpg');
+        }
+    }
+
+    ul.links {
+        clear: right;
+        .span(6);
+
+        li {
+            list-style-type: none;
+            margin-left: 0;
+            margin-bottom: @baseLine;
+            padding-bottom: 10px;
+            background: url(/media/img/mission/thin-stripe.png) center bottom repeat-x;
+            min-height: @baseLine * 4;
+        }
+
+        li:last-child {
+            padding: 0;
+            background-image: none;
+        }
+
+        h4 {
+            margin-bottom: @baseLine / 6;
+            a:after {
+                content: " »";
+                .open-sans-light;
+            }
+        }
+
+        p {
+            .font-size(.857em);
+            margin-bottom: @baseLine / 2;
+        }
+    }
+
+    li > ul {
+        margin-bottom: @baseLine;
+    }
+
+    .sidebar {
+        margin-top: 20px;
+    }
+}
+
+html[dir="rtl"] {
+    .content-main .intro {
+        float: right;
+    }
+}
+
+// Newsletter form
+#newsletter-subscribe {
+    background: transparent none;
+    color: @textColorPrimary;
+    margin-top: 0;
+
+    .form-title {
+        background: transparent none;
+        position: relative;
+        text-align: center;
+
+        h3 {
+            .font-size(28px);
+            color: @mozillaRed;
+            font-weight: bold;
+            line-height: 1.25;
+        }
+
+        h4 {
+            .font-size(18px);
+            .open-sans;
+            color: @textColorSecondary;
+            line-height: 1.25;
+        }
+
+        &:before {
+            background: @mozillaRed url('/media/img/newsletter/mozorg-newsletter.svg') center center no-repeat;
+            .background-size(100px, 100px);
+            border-radius: 100%;
+            content: '';
+            display: block;
+            height: 150px;
+            left: 0;
+            position: absolute;
+            top: 0;
+            width: 150px;
+        }
+    }
+
+    a:link,
+    a:visited {
+        color: @linkBlue;
+    }
+
+    a:hover,
+    a:focus {
+        color: @linkBlueHover;
+    }
+
+    input[type='email'] {
+        border: 1px solid @borderColor;
+    }
+
+    #newsletter-form-thankyou {
+        color: @textColorPrimary;
+
+        h3 {
+            color: @textColorPrimary;
+        }
+    }
+}
+
+
+
+/* Tablet Layout: 760px */
+@media only screen and (min-width: @breakTablet) and (max-width: @breakDesktop) {
+
+    #masthead,
+    .content,
+    .billboard {
+        width: @widthTablet - (@gridGutterWidth * 2);
+    }
+
+    .page-header {
+        background-image: url('/media/img/newsletter/intro-mozsummit-1000.jpg');
```

Comment:
Yes it's weird. This is derived from a much older file that didn't have a wide breakpoint, but I wanted to avoid totally refactoring the whole thing so I left the other queries alone. But perhaps just for the image swapping it'll be better to repeat the queries and have them together than to make this one odd standout. I shall revise.